packages/block-library/src/missing/index.js

```diff
@@ -16,7 +17,7 @@ export { metadata, name };
 
 export const settings = {
 	name,
-	title: __( 'Unrecognized Block' ),
+	title: __( 'Unrecognized' ),
```

Comment:
So I made this change to ensure the accessibility label doesn't read 'Unrecognized Block Block', but perhaps I could make the accessibility label instead read 'Block: Unrecognized Block' or similar.